device-types/Eaton/Tripp-Lite-PDUMH20HV.yml

```diff
@@ -0,0 +1,32 @@
+---
+manufacturer: Eaton
+model: Tripp Lite PDUMH20HV
+slug: eaton-tripp-lite-pdumh20hv
+part_number: 037332159014
+u_height: 1
+is_full_depth: false
```

Comment:
1. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. This appears to be listed in the datasheet (6.80 lbs or 3.08 kg)